is_triangle.py

```diff
@@ -0,0 +1,11 @@
+"""Is_Triangle."""
+
+
+def is_triangle(a, b, c):
+    """Test if given numbers can be valid sides of a triangle."""
+    if min([a, b, c]) <= 0:
+        return False
+    elif a + b <= c or a + c <= b or b + c <= a:
+        return False
+    else:
```

Comment:
Hi Will,

Thanks a ton for your comments.  This is the only one I don't understand.  The next line (not shown above) is "return True".  Are you sure this is not needed?  How else will the function return True?